roles/links-on-desktop/files/forallusers.desktop

```diff
@@ -0,0 +1,8 @@
+[Desktop Entry]
+Type=Application
+Encoding=UTF-8
+Exec=bash -c 'ln -s /adfc/links/ $(xdg-user-dir DESKTOP)/Links' 
```

Comment:
Schicke Idee

Reply:
Danke!
Georg wollte, dass webdav auf allen Stationen "installiert" wird. Da das Protokoll nativ unterstützt wird, reicht ein Link. 
Der Link zum zentralen Ordner ermöglicht es uns (IT) beliebige Links dort abzulegen.
Falls ein Benutzer der Meinung ist, dass er den Ordner nicht möchte und ihn einfach löscht, wird dieser beim nächsten Start einfach wieder erstellt. 

Der Teil mit $(xdg-user-dir DESKTOP) kam notgedrungen nach den ersten Tests hinzu (Desktop vs. Schreibtisch) ;)